test/e2e/case38_install_operator_test.go

```
@@ -362,11 +363,18 @@ var _ = Describe("Testing OperatorPolicy", Label("supports-hosted"), func() {
 	}
 
-	setupPolicy := func(ctx context.Context, opPolYAML, opPolName, parentPolicyName string) {
+	setupPolicy := func(ctx context.Context, opPolYAML, opPolName, parentPolicyName string, subNamespace ...string) {
 		GinkgoHelper()
 		patchedParentYAML := patchSingleField(ctx, parentPolicyYAML, testNamespace, "/metadata/name", parentPolicyName)
 		patchedOpPolYAML := parallelizeOpPol(ctx, opPolYAML, opPolName)
 
+		if len(subNamespace) > 0 {
+			patched := patchSingleField(
+				ctx, patchedOpPolYAML, testNamespace, "/spec/subscription/namespace", subNamespace[0])
+			os.Remove(patchedOpPolYAML)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)